### PR TITLE
Add a -v option to supply the package version

### PR DESCRIPTION
**Pull request by alanknight-wk**

### CODE REVIEW

**Comment by aviary3-wk** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://dev.workiva.net/docs/teams/information-security/aviary#configuring-aviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: lib/references_visitor.dart [hunk 1]
```diff
@@ -187,7 +187,10 @@ class AstReference {
       if (name is AstNode) {
         return name;
       }
-    } on NoSuchMethodError {} // ignore
+    } on NoSuchMethodError {
+      // ignore
+      null;
```
Comment (danchurch-wk) [inline lib/references_visitor.dart:192]: Should this be `return null`?

**Review by danchurch-wk — APPROVED** [review]
+1

**Comment by evanweible-wf** [comment]
Closing as stale for now. Feel free to re-open!

**Pull request closed without merging**